docs/reference/css_guidelines.rst

```diff
@@ -0,0 +1,151 @@
+Contributing CSS to Wagtail
```

Comment:
Maybe we should add a "contributing" section to the docs as our main contributing doc is getting quite big now.